jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/ui/utils/ViewExtensions.kt

```diff
@@ -0,0 +1,5 @@
+import android.content.Context
+import android.view.View
+
+val View.ctx: Context
```

Comment:
Consider deleting this. Extensions are nice, but aren't we happy just referencing this as `context`?